fusor-ember-cli/app/controllers/openstack/undercloud-deploy.js

```diff
@@ -56,102 +48,11 @@ const UndercloudDeployController = Ember.Controller.extend(NeedsDeploymentMixin,
       this.set('undercloudIP', null);
       this.set('sshUser', null);
       this.set('sshPassword', null);
-      this.set('isDeployed', false);
       this.set('openstackDeployment.undercloud_admin_password', null);
-    },
-
-    deployUndercloud() {
-      var self = this;
-      var openstackDeployment = this.get('openstackDeployment');
-      console.log('detectUndercloud');
-      console.log("host " + this.get('undercloudIP'));
-      console.log("user " + this.get('sshUser'));
-      var data = {
-        'underhost': this.get('undercloudIP'),
-        'underuser': this.get('sshUser'),
-        'underpass': this.get('sshPassword'),
-        'deployment_id': this.get('deploymentId')
-      };
-
-      var promiseFunction = function (resolve) {
```

Comment:
I refactored this promise and put it in the route to share some code and simplify it a little.